yew-macro/src/html_tree/html_tag/mod.rs

```diff
@@ -140,7 +140,7 @@ impl ToTokens for HtmlTag {
         let attr_pairs = attributes.iter().map(|TagAttribute { label, value }| {
             let label_str = label.to_string();
             if label.optional.is_some() {
-                quote_spanned! {value.span()=> (#label_str.to_owned(), if let Some(__yew_value) = #value { Some((__yew_value).to_string()) } else { None }) }
+                quote_spanned! {value.span() => (#label_str.to_owned(), if let Some(__yew_value) = #value { Some((__yew_value).to_string()) } else { None }) }
```

Comment:
Please take a look at the documentation of the `quote_spanned!` macro. It states the following:
> The lack of space before the => should look jarring to Rust programmers and this is intentional. The formatting is designed to be visibly off-balance and draw the eye a particular way, due to the span expression being evaluated in the context of the procedural macro and the remaining tokens being evaluated in the generated code.

Reply:
Strange to put whitespace rules in an environment where whitespace is ignored, but okay.